profile/README.md

```diff
@@ -0,0 +1,30 @@
+<p align="center">
+    <img width="400px" src="https://raw.githubusercontent.com/FluxML/Flux.jl/master/docs/src/assets/logo.png"/>
+</p>
+
+# The Elegant Machine Learning Stack
+
+Flux is a 100% pure-Julia stack and provides lightweight abstractions on top of Julia's native GPU and AD support. It makes the easy things easy while remaining fully hackable.
+
+## Features
+
+Flux has features that sets it apart among ML systems
+
+### Compiled Eager Code
+Flux provides a single, intuitive way to define models, just like mathematical notation. Julia transparently [compiles your code](https://julialang.org/blog/2018/12/ml-language-compiler), optimising and fusing kernels for the GPU, for the best performance.
+
+### Differentiable Programming
+Existing Julia libraries are differentiable and can be incorporated directly into Flux models. Cutting edge models such as [Neural ODEs](https://julialang.org/blog/2019/01/fluxdiffeq) are first class, and [Zygote](https://github.com/FluxML/Zygote.jl) enables overhead-free gradients.
```

Comment:
There should be a more up-to-date description of DiffEqFlux over on the SciML site.